scripts/serie_update.py

```diff
@@ -355,15 +345,8 @@ def build_patch_from_current_zephyr_version(self):
         hal_conf_patch = self.stm32cube_temp / "hal_conf.patch"
         if hal_conf.exists():
             os_cmd(
-                (
-                    "git",
-                    "diff",
-                    "HEAD@{1}",
-                    "--",
-                    str(hal_conf),
-                    ">>",
-                    str(hal_conf_patch),
-                ),
+                "git diff HEAD@{1} -- " + str(hal_conf)
```

Comment:
in general it is better to use multi-line strings if they do not fit in one line, so I prefer the previous code in this case, others LGTM

Reply:
Unfortunately, except the comment in line 33, this is the minimal set of changes to get it running.
The arguments are treated differently on unix systems than on windows.

[Popen Constructor](https://docs.python.org/2/library/subprocess.html#popen-constructor):
_"On Unix with shell=True,   . If args is a sequence, the first item specifies the command string, and any additional items will be treated as additional arguments to the shell itself"_

In this case "git" is executed without arguments, and the rest of the sequence are arguments passed to the shell.
I am afraid that there is no way on unix systems to avoid passing the whole command as a string for commands that need "shell=True". But if you know any better solution that could work I will try it.

Reply:
@gmarull I confirm this is working. Is that blocking ?